src/components/Drawer/Drawer.js

```diff
@@ -49,14 +50,27 @@ const Drawer = props => {
         return STATUS_LIST.IS_HIDDEN;
     }
 
-    function handleClick() {
+    function triggeredIsExpanded() {
```

Comment:
maybe call it `toggleDrawer` or `handleToggleCall` or `toggleIsExpanded`. The current name is not clear

Reply:
Good point!